src/main/java/bio/overture/ego/service/TokenService.java

```diff
@@ -337,6 +333,15 @@ public Claims getTokenClaims(String token) {
     }
   }
 
+  public Claims getTokenClaimsIgnoreExpiry(String token) {
+    try {
+      return getTokenClaims(token);
+    } catch (ExpiredJwtException exception) {
+      log.info("Refreshing expired token!", exception);
```

Comment:
Let's avoid printing the stacktrace everytime this happens due to the `info` log level. Let's put the stack trace into `debug` while having the `info` print a single line. I've given an example here with some guards:
```java
  public Claims getTokenClaimsIgnoreExpiry(String token) {
    try {
      return getTokenClaims(token);
    } catch (ExpiredJwtException exception) {
      val claims = exception.getClaims();
      if (StringUtils.isEmpty(claims.getId())) {
        throw SomeSortofIllegalStateException
      }
      log.info("Refreshing expired token: {}", claims.getId());
      log.debug("Refreshing expired token!", exception);
      return claims;
    }
  }
```